test/unit_tests/iovec.cpp

```diff
@@ -0,0 +1,45 @@
+#include <algorithm>
+#include <vector>
+
+#include "oc_ansi.h"
+
+#include <catch2/catch.hpp>
+
+// This function is the one that is used in all nrn-modeldb-ci
+// Keep as is
+int cmpdfn(double a, double b) {
+    return ((a) <= (b)) ? (((a) == (b)) ? 0 : -1) : 1;
```

Comment:
```suggestion
    return (a <= b) ? ((a == b) ? 0 : -1) : 1;
```

Reply:
This is a copy/paste how the usage of this function ^^

Do you think we should still fix it?